lms/djangoapps/teams/api.py

```diff
@@ -62,6 +62,14 @@ def is_team_discussion_private(team):
     return getattr(team, 'is_discussion_private', False)
 
 
+def is_instructor_managed_team(team):
+    """
+    Return true if the team is managed by instructors.
+    For now always return false, will complete the logic later.
+    """
+    return False
```

Comment:
I just updated https://openedx.atlassian.net/browse/MST-25 to include implementing this function.

Could you include the text `TODO MST-25` in the docstring?